packages/fxa-auth-server/test/local/senders/mjml-emails.ts

```diff
@@ -1543,6 +1744,597 @@ describe('lib/senders/mjml-emails:', () => {
+            emailService: 'fxa-auth-server',
+            emailSender: 'ses',
+            mailer: mailer.mailer,
+          },
+        ])
+      );
+
+      return mailer.send(message).then(() => {
+        assert.equal(mailer.selectEmailServices.callCount, 1);
+        assert.equal(mailer.emailService.sendMail.callCount, 2);
+        assert.equal(mailer.mailer.sendMail.callCount, 1);
+
+        let args = mailer.emailService.sendMail.args[0];
+        assert.equal(args.length, 2);
+        assert.equal(args[0].to, 'foo@example.com');
+        assert.equal(args[0].cc, undefined);
+        assert.equal(args[0].provider, 'sendgrid');
+
+        let headers = args[0].headers;
+        assert.equal(headers['X-Email-Service'], 'fxa-email-service');
+        assert.equal(headers['X-Email-Sender'], 'sendgrid');
+
+        args = mailer.emailService.sendMail.args[1];
+        assert.equal(args.length, 2);
+        assert.equal(args[0].to, 'bar@example.com');
+        assert.equal(args[0].cc, undefined);
+        assert.equal(args[0].provider, 'ses');
+
+        headers = args[0].headers;
+        assert.equal(headers['X-Email-Service'], 'fxa-email-service');
+        assert.equal(headers['X-Email-Sender'], 'ses');
+
+        args = mailer.mailer.sendMail.args[0];
+        assert.equal(args.length, 2);
+        assert.equal(args[0].to, 'baz@example.com');
+        assert.equal(args[0].cc, undefined);
+        assert.equal(args[0].provider, undefined);
+
+        headers = args[0].headers;
+        assert.equal(headers['X-Email-Service'], 'fxa-auth-server');
+        assert.equal(headers['X-Email-Sender'], 'ses');
+      });
+    });
+  });
+});
+
+describe('mailer constructor:', () => {
+  let mailerConfig, mockLog, mailer;
+
+  before(async () => {
+    mailerConfig = [
+      'accountSettingsUrl',
+      'accountRecoveryCodesUrl',
+      'androidUrl',
+      'initiatePasswordChangeUrl',
+      'initiatePasswordResetUrl',
+      'iosUrl',
+      'iosAdjustUrl',
+      'passwordManagerInfoUrl',
+      'passwordResetUrl',
+      'privacyUrl',
+      'reportSignInUrl',
+      'sender',
+      'sesConfigurationSet',
+      'supportUrl',
+      'syncUrl',
+      'verificationUrl',
+      'verifyLoginUrl',
+      'verifySecondaryEmailUrl',
+      'verifyPrimaryEmailUrl',
+    ].reduce((target, key) => {
+      target[key] = `mock ${key}`;
+      return target;
+    }, {});
+    mockLog = mocks.mockLog();
+    mailer = await setup(
+      mockLog,
+      { ...config, smtp: mailerConfig },
+      {},
+      'en',
+      'wibble'
+    );
+  });
+
+  it('mailer and emailService are both mocked', () => {
+    assert.equal(mailer.mailer, 'wibble');
+    assert.equal(mailer.emailService, 'wibble');
+  });
+
+  it('set properties on self from config correctly', () => {
+    Object.entries(mailerConfig).forEach(([key, expected]) => {
+      assert.equal(mailer[key], expected, `${key} was correct`);
+    });
+  });
+});
+
+describe('email translations', () => {
```

Comment:
I think we should remove these for now and anything else referencing "localization" added here - we purposefully have kept those out since it's handled differently with the new emails. We'll have Fluent translations tested when we get #11529 in, and the `_getLocalizedCurrencyString` should be handled when #11471 is in.